docs/en/guides/contribution/How-to-release.md

```diff
@@ -6,7 +6,10 @@ This documentation guides the release manager to release the SkyWalking Satellit
 
 1. Close(if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-satellite](https://github.com/apache/skywalking-satellite/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
 2. Update [CHANGES.md](../../../../CHANGES.md).
-
+3. Generate LICENSE file in [`dist/LICENSE`](../../../../dist/LICENSE) and copy dependencies' licneses to [`dist/licenses`](../../../../dist/licenses/) folder, this can be done by running the following command in the project root:
+```shell
+license-eye dep resolve --summary dist/LICENSE.tpl -o dist/licenses
```

Comment:
Should we consider to locking eyes version?
I am asking this, because I want this step in oap and Java agent too.

Reply:
Added instruction to install specific version of license-eye